package.json

```diff
@@ -22,7 +22,7 @@
     "closure": "cd lighthouse-core && closure/closure-type-checking.js",
     "watch": "lighthouse-core/scripts/run-mocha.sh --watch",
     "chrome": "lighthouse-core/scripts/launch-chrome.sh",
-    "dbw": "npm run start -- --config-path=lighthouse-core/config/dobetterweb.json --disable-device-emulation --disable-cpu-throttling --disable-network-throttling --output=html --output-path=results.html",
+    "dbw": "npm run start -- --disable-device-emulation --disable-cpu-throttling --disable-network-throttling --output=html --output-path=results.html",
```

Comment:
if we're going to get rid of the dbw-only config, this should be dropped (default already covers everything here). If it's just for the dropping of emulation, we could rename this `fast` or something